src/main/java/com/github/bartimaeusnek/cropspp/ConfigValues.java

```diff
@@ -13,4 +13,5 @@
     public static float TConstructBerryGain = 1;
     public static float PrimordialBerryGain = (float) 0.5;
     public static boolean WiPItems = false;
+    public static boolean ILoveScreaming = false;
```

Comment:
I agree should default true otherwise nobody gets to experience the fun

And then can turn it off.

Reply:
or even better just let it be = the sound to scream ....